src/block-components/progress-bar/style.js

```diff
@@ -1,43 +1,49 @@
-import { useStyles, getStyles } from '~stackable/util'
+import {
+	useStyles, getStyles, hexToRgba,
+} from '~stackable/util'
 import { Style as StyleComponent } from '~stackable/components'
 import {
 	DEFAULT_PERCENT, DEFAULT_SIZE, DEFAULT_THICKNESS,
 } from './attributes'
 
-const getStyleParams = () => {
+const getStyleParams = ( { isCircle } ) => {
 	return [
 		{
-			selector: '.stk-progress-circle',
-			styleRule: '--progress-thickness',
-			attrName: 'progressThickness',
-			format: '%spx',
+			selector: '[class*="stk-progress-"]',
+			styleRule: '--progress-percent',
+			attrName: 'progressPercent',
+			...( ! isCircle && {
+				format: '%s%',
+			} ),
 		},
 		{
 			selector: '.stk-progress-circle',
 			styleRule: '--progress-color-1',
 			attrName: 'progressColor1',
-			valuePreCallback: ( value, getAttribute, _, state ) => {
-				if ( getAttribute( 'progressColorType', 'desktop', state ) === 'gradient' ) {
+			valuePreCallback: ( value, getAttribute ) => {
+				if ( getAttribute( 'progressColorType' ) === 'gradient' && isCircle ) {
```

Comment:
Seems that this style is only applicable for when `isCircle` is `true`? If yes, then I recommend using only adding this entry only for circles